docs/01-overview/main-areas/observability/README.md

```diff
@@ -8,23 +8,23 @@ Out of the box, Kyma provides tools to collect and ship telemetry data using the
 
 Kyma collects telemetry data with the following in-cluster components:
 
-- [Prometheus](https://prometheus.io/docs/introduction) collects metrics from Pods. Metrics are the time-stamped data that provide information on the running jobs, workload, CPU consumption, memory usage, and more. All metrics relevant for observing the in-cluster Istio Service Mesh are collected separately.
-
 - [Fluent Bit](https://fluentbit.io/) collects logs, provided using the [Telemetry Module](../telemetry/README.md).
 
 - An [OTel Collector](https://opentelemetry.io/docs/collector/) collects traces, provided using the [Telemetry Module](../telemetry/README.md).
 
 The collected telemetry data are exposed so that you can view and analyze them with observability tools.
 
-> **NOTE:** Kyma's [telemetry component](../telemetry/README.md) supports providing your own output configuration for application logs and traces. With this, you can connect your own observability systems inside or outside the Kyma cluster with the Kyma backend.
+> **NOTE:** Kyma's [telemetry component](../telemetry/README.md) supports providing your own output configuration for application's logs, traces and metrics. With this, you can connect your own observability systems inside or outside the Kyma cluster with the Kyma backend.
```

Comment:
The telemetry component does not support metrics yet. That's just the plan for the future.